.prettierignore

```diff
@@ -11,5 +11,5 @@ expected-cdk-out
 .changeset/pre.json
 concurrent_workspace_script_cache.json
 scripts/components/api-changes-validator/test-resources/working-directory
-test-projects
+/test-projects
```

Comment:
ignore the root test-projects dir but format the test projects in the integration-tests folder